### PR TITLE
configurable powTarget and powTime

### PR DESCRIPTION
**Pull request by rasom**
status: ready

### CODE REVIEW

File: src/status_im/protocol/web3/delivery.cljs [hunk 1]
```diff
@@ -242,9 +243,11 @@
           ;; check each message asynchronously
           (when (should-be-retransmitted? options data)
             (try
-              (let [message' (check-ttl message type ttl-config default-ttl)
-                    callback (delivery-callback web3 post-error-callback data message')]
-                (t/post-message! web3 message' callback))
+              (let [message'  (check-ttl message type ttl-config default-ttl)
+                    message'' (assoc message' :powTarget pow-target
+                                              :powTime pow-time)
```
Comment (jeluard) [inline src/status_im/protocol/web3/delivery.cljs]: Any chance we can avoid `message''` ? :)

**Review by jeluard — APPROVED** [review]

**Comment by rasom** [comment]
https://status-im.slack.com/archives/C9B1T4B9B/p1520577085000029
@JekaMas :

> Hey guys! I was thinking about testing Status app with default minPoW=0.2 instead of current 0.01. May be it makes sence to make such test version with whisperV5 to test does messages more reliable in whole Ethereum network?

File: src/status_im/protocol/web3/delivery.cljs (hunk 1 above)
Comment (rasom) [reply src/status_im/protocol/web3/delivery.cljs]: yes sure

Comment (rasom) [reply src/status_im/protocol/web3/delivery.cljs]: done

**Review by flexsurfer — APPROVED** [review]

**Comment by asemiankevich** [comment]
@rasom @flexsurfer any functional changes to the app or anything to test there? Sorry a bit confused

**Comment by rasom** [comment]
@JekaMas did you have a chance to check this PR?

**Comment by asemiankevich** [comment]
i have checked that i am still able to exchange messages within this PR. dont see any ethereum nodes in geth log for android (boot nodes) in this PR when messaging though. Not sure if this expected
[geth.log](https://github.com/status-im/status-react/files/1830103/geth.log)

**Comment by asemiankevich** [comment]
@JekaMas please take a look

**Comment by JekaMas** [comment]
@asemiankevich At this step, we only expect the usual Status boot nodes. We only can test whisperV5 application because there are no any whisperV6 nodes in the world except Status :-)

**Comment by asemiankevich** [comment]
okay cool :) Thanks @JekaMas :)

**Pull request merged**